libs/ngrx-accelerator/src/lib/store-connector/onecx-state.ts

```diff
@@ -0,0 +1,6 @@
+import { PayloadNavigatedEvent } from '@onecx/integration-interface'
+
+export type LocationState = PayloadNavigatedEvent
```

Comment:
I guess it's also fine doing it like that, but it might be an idea to have PayloadNavigatedEvent have LocationState as an attribute instead? Something like `PayloadNavigatedEvent  { newState: LocationState }`

Reply:
see PM